### DISCUSSION TITLE
Client's screen lock doesn't work on two monitors system

### DISCUSSION THREAD

**Discussion by tauhidbd2007** [epoptes/epoptes#35] [Q&A]
In my lab, the computer's are configured as one system with two monitors display. When I try to lock any particular client's screen from epoptes server, it just able to lock only one monitor, not both. In other cases like broadcasting, monitoring or taking over client control, both monitor screens are displayed. Is there any solution for this issue?

**Comment by alkisg** [comment]
The older 0.5.10 Epoptes version is based on Python2/Gtk2 and will not receive any updates.

I uploaded the newer 1.0.0 version in the [Epoptes proposed PPA](http://www.epoptes.org/documentation/ppa) for Ubuntu 18.10, 18.04 and 16.04.
If you still have the problem with that version, I could take a look.

**Comment by alkisg** [comment]
No reply in 2 months, closing this issue.
